addons/mail/static/src/models/composer/composer.js

```diff
@@ -567,6 +564,9 @@ function factory(dependencies) {
         isDoFocus: attr({
             default: false,
         }),
+        isLog: attr({
+            default: false,
+        }),
```

Comment:
missing comment (looks obvious in scope of this PR, but it's nice to avoid surges of incomprehension for our good health)